src/transactions/simulation/SimulationTransactionFrame.cpp

```diff
@@ -100,4 +100,19 @@ SimulationTransactionFrame::processFeeSeqNum(AbstractLedgerTxn& ltx,
         acc.seqNum = mEnvelope.tx.seqNum;
     }
 }
+
+void
+SimulationTransactionFrame::processSeqNum(AbstractLedgerTxn& ltx)
+{
+    auto header = ltx.loadHeader();
+    if (header.current().ledgerVersion >= 10)
+    {
+        auto sourceAccount = loadSourceAccount(ltx, header);
+        if (sourceAccount.current().data.account().seqNum > mEnvelope.tx.seqNum)
+        {
+            return;
+        }
+        sourceAccount.current().data.account().seqNum = mEnvelope.tx.seqNum;
```

Comment:
I think lines 111-114 should be removed, making this analogous to lines 98-101.

Reply:
Done.